packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -3907,3 +3907,15 @@ it.each(['positive', 'warning', 'negative'] as const)(
     expect(rows[1]).not.toHaveAttribute('data-iui-status', rowStatus);
   },
 );
+
+it('should navigate through table sorting with the keyboard', async () => {
+  const onSort = jest.fn();
+  renderComponent({
+    isSortable: true,
+    onSort,
+  });
+
+  await userEvent.tab(); // tab to sort icon button
+  await userEvent.keyboard('{Enter}');
+  expect(onSort).toHaveBeenCalled();
```

Comment:
What about filtering?

Reply:
I was having lots of trouble with the filtering test and couldn't make it fail when it was supposed to. Do you have any suggestions? https://teams.microsoft.com/l/message/19:23e76cd5984f40eca958bc496bb00aaa@thread.tacv2/1661891083535?tenantId=067e9632-ea4c-4ed9-9e6d-e294956e284b&groupId=8a0b1bff-5d9d-4a29-9112-7d412098d768&parentMessageId=1661891083535&teamName=iTwinUI%20development&channelName=Issues%20and%20help&createdTime=1661891083535&allowXTenantAccess=false

Reply:
Have you tried `fireEvent.focus(el)`?

Reply:
Okay, I tried:
`const filterIconButton = container.querySelector(
    '.iui-table-filter-button',
  ) as HTMLElement;`
  `fireEvent.focus(filterIconButton);`
and
  `fireEvent.blur(filterIconButton);`
and
  `filterIconButton.focus();`
and
  `const table = container.querySelector('.iui-table') as HTMLElement;`
  `fireEvent.keyDown(table, { key: 'Tab' });`

Only `blur` passes-ish, but it passes every time and it gives a console error:
![Screenshot 2022-09-09 082108](https://user-images.githubusercontent.com/37936169/189359693-60dec981-5d51-46dd-832d-f48e888103d2.png)
I'm in the process of looking the error up but haven't found anything yet.

Reply:
Readded filter test